README.md

```diff
@@ -4,6 +4,11 @@ Follow [https://www.mkdocs.org/getting-started/](https://www.mkdocs.org/getting-
 & [https://squidfunk.github.io/mkdocs-material/getting-started/](https://squidfunk.github.io/mkdocs-material/getting-started/)
 
 Run : 
-- `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
-- `mkdocs serve` in the project root folder
-- Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via Python
+   - `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
+   - `mkdocs serve` in the project root folder
+   - Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via [Docker](https://squidfunk.github.io/mkdocs-material/getting-started/ "more recent methods and docs may very well be available there") (in case you are not a big python fan)
+   - build an image from de provided [Dockerfile](./Dockerfile "you'll only need to do this once") `docker build -t squidfunk/mkdocs-material .`
```

Comment:
In order to not confuse with squidfunk/mkdocs-material original image, we can choose an other name for our custom image: `docker build -t chutney/doc .`

Reply:
I must say on this one I fully went like what mkDocs's documentation recommends (using the same image name and so on)
Tho I get the idea of having your very own image name for clarity's sake :)